tests/serializers/test_any.py

```diff
@@ -201,6 +201,30 @@ def test_any_config_timedelta_float_faction():
     assert s.to_json({one_half_s: 'foo'}) == b'{"1.5":"foo"}'
 
 
+def test_any_config_timedelta_millisecond():
```

Comment:
Looks great, thanks for adding tests. We could probably parametrize with the above tests to simplify the repeated logic?

Can you add some tests for negative behavior as well?

Reply:
I think parametrising could have the opposite effect and obscure the logic by the amount of variables passed into the test?

Negative behaviour's a good idea though 👍🏻

Reply:
> I think parametrising could have the opposite effect and obscure the logic by the amount of variables passed into the test?

Fair enough, will leave the test design up to you.